src/Zicht/Bundle/UrlBundle/Url/ShortUrlGeneratorInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright Zicht Online <http://zicht.nl>
+ */
+
+namespace Zicht\Bundle\UrlBundle\Url;
+
+interface ShortUrlGeneratorInterface
+{
+    /**
+     * @param string $url
+     * @param int $length
+     * @return string
+     */
+    public function generate($url, $length = 8);
```

Comment:
Zou geen 'default' waarde gebruiken.  Het wordt nu altijd met een waarde aangeroepen, die 'toevallig' ook 8 is.  Daardoor staat nu 3x een 8 in de code, waarvan er 2 niets doen

Reply:
Ja, is een beetje een code-smell geworden. Je kunt dit getal ook nauwelijks beinvloeden.

Reply:
Hmmm nee die length is nodig om een collision te voorkomen. Als er een collision is dan maken we de hash 1 karakter langer, net zolang tot er geen collision meer is. Maar ik vraag me af of het uberhaupt wel nodig is...